models/Recording.js

```diff
@@ -106,6 +107,41 @@ module.exports = function(sequelize, DataTypes) {
     return await this.findOne(query);
   };
 
+  /**
+   * Deletes a single recording if the user has permission to do so.
+   */
+  var deleteOne = async function(user, id) {
+    var recording = await this.getOne(user, id);
+    if (recording == null)
+      return false;
+    var userPermissions = await recording.getUserPermissions(user);
+    if (userPermissions.canDelete != true)
+      return false;
+    else {
```

Comment:
Not using curly brackets is technically fine but it can end up leading to hard to find bugs.

https://stackoverflow.com/a/4797368/195383

A lot of places have a style guide which says you must always used braces even when not necessary and I tend to lean that way too.

Reply:
I am guessing we can/should add that to the .eslintrc.yml file then?

Reply:
Yes, but let's do that separately

Reply:
Actually, this was dead easy to do so I added it :)